src/Controller/OrderController.php

```diff
@@ -390,24 +388,7 @@ public function paymentAction(Request $request,
 
             $this->objectManager->flush();
 
-            if (PaymentInterface::STATE_FAILED === $payment->getState()) {
-
-                $error = $payment->getLastError();
-
-                // Make sure to retrieve the last payment
-                $payment = $order->getLastPayment(PaymentInterface::STATE_CART);
-
-                // Make sure to call StripeManager::configurePayment()
-                // It will resolve the Stripe account that will be used
-                // TODO Make sure we are using Stripe, not MercadoPago
-                $stripeManager->configurePayment($payment);
-
-                return $this->render('order/payment.html.twig', array_merge($parameters, [
-                    'form' => $form->createView(),
-                    'error' => $error,
-                    'payment' => $payment,
-                ]));
-            }
+            // TODO Check if all $payments have succeeded
```

Comment:
needs to be done before merging no?
at least port the old stripe code here if it is a stripe payment ?